app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss

```diff
@@ -61,6 +61,7 @@
 
   &__content{
     @include absolute-center;
+    -webkit-transform: translateX(-50%) translateY(-50%);
```

Comment:
Not sure where absolute-center comes from but if the only goal here is the vertically align then foundation has an include that does the transform as well: vertical-center